nightfall-optimist/src/services/database.mjs

```diff
@@ -466,7 +474,8 @@ export async function saveTree(blockNumber, blockNumberL2, timber) {
   const connection = await mongo.connection(MONGO_URL);
   const db = connection.db(OPTIMIST_DB);
   return db.collection(TIMBER_COLLECTION).insertOne({
-    _id: blockNumber,
+    _id: timber.root,
```

Comment:
we'll need to re-key this with something under than `root`. A block full of withdrawals will have the same root and throw a duplicate error. Perhaps `blockNumberL2` will be better?

Reply:
Oh blast.  I forgot about the dastardly 'block of withdrawals'.